spec/models/certificate_spec.rb

```diff
@@ -235,6 +235,22 @@
         ].join("\n\n")
       end
     end
+
+    describe '#signing_key_id' do
+      it 'handles multiline authorityKeyIdentifier' do
+        # stub multiline aKI
+        expect(certificate).to receive(:get_extension).with('authorityKeyIdentifier').and_return(
```

Comment:
This is fine for the function. I'll try to create a test (later) that can create a certificate with all of this without any mocking. Based on the RFC, I should be able to.

Reply:
`easy-rsa` from OpenVPN generates certificates like these by default, so I can give some fully working examples if that's helpful.